.travis.yml

```diff
@@ -5,7 +5,6 @@ env:
   - TF_CPP_MIN_LOG_LEVEL=2
 install:
   - pip install nose
-  - pip install "tensorflow==2.3"
   - pip install "pillow>=7.2,<7.3"
   - pip install "scikit-learn>=0.23,<0.24"
   - pip install . -v
```

Comment:
This is still the right command, yeah?  Invoking pip the same way?

Reply:
I ran that command in a clean virtualenv, and it installed properly. Looks like Travis is unhappy with the log length though, so maybe the -v needs to go.